users.go

```diff
@@ -246,7 +243,7 @@ func (api *Client) GetUserIdentityContext(ctx context.Context) (*UserIdentityRes
 
 // SetUserPhoto changes the currently authenticated user's profile image
 func (api *Client) SetUserPhoto(ctx context.Context, image string, params UserSetPhotoParams) error {
-	return api.SetUserPhoto(context.Background(), image, params)
```

Comment:
This is definitely broken, anyone who currently calls this method will get an infinite loop. For consistency the API *should* be:

```
SetUserPhoto(image string, params UserSetPhotoParams)
```

but that would be a breaking change.

Reply:
I think we can safely assume nobody is using this, so please go ahead and make the suggested change. After that this looks ready to merge imo. @catherinetcai agree?

Reply:
Done